src/config.ts

```diff
@@ -1,15 +1,45 @@
 import { makeConfig } from 'cleaner-config'
-import { asObject, asOptional, asString } from 'cleaners'
+import { asNumber, asObject, asOptional, asString } from 'cleaners'
+
+// Custom cleaners:
+
+const asNumberString = (raw: unknown): string => {
+  const numStr = asString(raw)
+  if (!/^\d+(?:.\d+)$/.test(numStr)) {
+    throw TypeError('Expected number string')
+  }
+  return numStr
```

Comment:
You should do it as:

 ```js
const clean = Number(asString(raw))
if (isNaN(clean)) throw new TypeError('Expected number string')
return clean
```

That way the result is a proper number. I already had to fix the bug that `httpPort` below was a string and not a number when I added @types/express to the repo.

Reply:
From our discussion: no need for the change because type transformation is handled outside of the cleaner in this context (line 39 for `httpPort`)